libdino/src/service/blocking_manager.vala

```diff
@@ -20,19 +20,22 @@ public class BlockingManager : StreamInteractionModule, Object {
         this.stream_interactor = stream_interactor;
     }
 
-    public bool is_blocked(Account account, Jid jid) {
+    public bool is_blocked(Account account, Jid jid, bool domainblock = false) {
```

Comment:
Instead of adding a `domainblock` parameter here, you could just pass the domain JID, when a domain block is desired. The `Jid` object has a property `domain_jid`.

Reply:
Oh I didn't remember this, thanks for pointing it out.